scripts/install

```diff
@@ -66,19 +66,40 @@ port=$(ynh_find_port 3000)	# Cherche un port libre.
 ynh_app_setting_set $app port $port
 
 #=================================================
-# INSTALL DEPENDENCIES
+# INSTALL NODEJS
 #=================================================
+ynh_install_nodejs 8.0.0
```

Comment:
They say the latest version. I just choose 8.0.0 but indeed, it's better to choose 8.2.1